server/fleet/host_certificates.go

```diff
@@ -143,3 +142,10 @@ func ExtractDetailsFromOsqueryDistinguishedName(str string) (*HostCertificateNam
 	// TODO
 	return nil, errors.New("not implemented")
 }
+
+func firstOrEmpty(s []string) string {
```

Comment:
Just a thought - this could be made generic and added in the package along this similar slice helper: https://github.com/fleetdm/fleet/blob/606df3f34913cd2e4edeb11bbaf7595bdf92a31b/server/utils.go#L177